tools/pipelines/build-docs-website-deploy.yml

```
@@ -0,0 +1,110 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# build-docs-website-deploy pipeline
+# This pipeline triggers on the completion of the build-docs pipeline (main branch run).
+# After the build-docs pipeline completes, this pipeline downloads the generated fluidframework-docs
+# artifacts and deploys the content using AzureStaticWebApp.
+# This pipeline also has a scheduled trigger which runs every 4 hours.
+# Note that this pipeline contains two triggers: a scheduled trigger which captures changes made in
+# the release branches and a pipeline completion trigger which captures the changes on main.
+# Ideally, changes to both branches would be captured in pipeline completion triggers, however,
+# due to limitations with pipeline version configurations. It seems like its not possible to set 
+# "release/*" branch as a pipeline trigger while running this pipeline using the "main" branch version.
+name: $(Build.BuildId)
+
+schedules:
+  - cron: "0 */4 * * *"
+    displayName: Website deploy with 4 hour increments
```

Nit: since this is what will be displayed in the ADO pipeline feed, to help differentiate run triggers, it might be better to display something like:
```suggestion
    displayName: Scheduled deployment
```